[DOCS-12] getting started dedicated branch

mjang-cobalt opened the pull request:
You can review the build here: https://draft2-getting-started.netlify.app/getting-started/ (courtesy of Netlify CI).

I now realize that the size of this PR (100 files) might make it difficult for reviewers. I can walk you though how to find which file corresponds to what page.

Alternatively, if you don't want to comment in the PR directly, I suggest using https://hypothes.is

- Follow [these instructions](https://web.hypothes.is/start/)
- Highlight text
- You should see pop-up options similar to:

<img width="306" alt="Screen Shot 2021-08-24 at 1 18 33 PM" src="https://user-images.githubusercontent.com/84352037/130684090-0b15f800-187e-4e18-b917-3c301e0e70d1.png">

Highlight annotate and describe your concerns / suggested change.

Since the repo is public, it's OK to make comments, and then select "Post to Public":

<img width="392" alt="Screen Shot 2021-08-24 at 1 23 06 PM" src="https://user-images.githubusercontent.com/84352037/130684681-13344cf8-c469-4d2a-8528-b9907c517a51.png">

reginandrade commented on content/en/Getting started/Assets/asset-description.md:
```diff
@@ -0,0 +1,119 @@
+Help us find the right pentesters for your asset. Include a high-level overview
+of the application. Add details such as:
+
+- Language, such as JavaScript or Ruby on Rails.
```
I wonder if we should just leave this general, as "Coding languages" - given that almost all web apps have some flavor of javascript?

mjang-cobalt replied:
Point well taken. "JavaScript" is sort of like "Kleenex" at this point.

reginandrade commented on content/en/Getting started/planning.md:
```diff
@@ -0,0 +1,31 @@
+{{% /pageinfo %}}
+
+By default, we can help you schedule pentests with a start date at least
+48 hours after you select "Submit for Review".
```
We note the start date limited to "2 business days" in the application, which I believe is in line with our staffing process.  For example, there is form validation in the wizard Planning step: "The pentest start date needs to be at least 2 business days from today,"

mjang-cobalt replied:
Yes, for consistency with language in the UI (when it's clear)

reginandrade commented on content/en/Getting started/planning.md:
```diff
@@ -0,0 +1,31 @@
+On the Pentest Planning screen, you can also update the asset scope, and
+therefore the number of credits required for the test.
+
+You can also specify special requirements for pentesters. For example, if
```
I think the special requirements section is no longer in the wizard, and we are only enabling it through the Cobalt admin console. Not sure because there is also a feature flag involved.  There were recent code changes to remove it, you could verify with @marijanamustra and EX team.

marijanamustra replied:
Hi, removing the requirements section from the wizard is under a feature flag so not out yet. The plan is that adding requirements will be available to CSMs in editing brief (and in admin under individual postings on the pentest). You can see more here https://zombie.atlassian.net/browse/EX-1183 or if you need more information, I would suggest posting a question on [EX channel](https://cobalthq.slack.com/archives/CTSGKCXAL) Thank you

mjang-cobalt replied:
Thanks for the update. I did not know of the plan, and will monitor EX-1183. However, I need to reflect the Wizard as it exists today.

reginandrade commented on content/en/Getting started/sign-in.md:
```diff
@@ -0,0 +1,55 @@
+   - One (1) uppercase letter
+   - One (1) lowercase letter
+   - One (1) digit
+
```
one other requirement, which i'm not sure we need to mention, is that you cannot use any of the last 10 previous passwords.  specifying that in the docs might provide more information than what is really needed, and give clues to anyone attempting to hack our authentication?

mjang-cobalt replied:
I'm tempted to include a note... but I'd like to find the code file to watch (my search foo failed me here). @reginandrade can you point me to the code that specifies "last 10 previous passwords"

reginandrade commented on content/en/Getting started/sign-in.md:
```diff
@@ -0,0 +1,55 @@
+
+Even if you haven't yet purchased Cobalt credits, this page (and document)
+can help you visualize how you can set up a pentest with the Cobalt UI.
+
```
Do we need to mention anything about sign in when an organization has SAML or Azure SSO, that they will use that provider instead of email+password on Cobalt sign in page?

mjang-cobalt replied:
I have to think about this...

Since this is a "Getting Started Guide", I'm defaulting to "no". However, in plans for future docs, I don't have an "Obvious" place for our SAML/3rd Party Sign-in options.

mjang-cobalt replied:
Investigating further, it (now) bothers me that we have "Identity & Access" in a different area than 2FA. Gosh.

In any case, I'm now going to expand the note at the end of the [Sign In To Cobalt](https://draft2-getting-started.netlify.app/getting-started/sign-in/) page.

![Screen Shot 2021-09-14 at 12 11 35 PM](https://user-images.githubusercontent.com/84352037/133319635-882311e0-b9de-463a-96a7-0c7aca270aa7.png)

mjang-cobalt commented:
Thanks @reginandrade and @marijanamustra for your comments. If you're OK with my changes, I'd appreciate your "approval".

mjang-cobalt commented:
Note: this has been reviewed by numerous parties, including 

@reginandrade
Jacob Fox
Vasilena Stamboliyska
@TheInstaGrahame 
@penelopeyao 
@sam-cobalt 
@dannyleong39 

Thanks so much! I hope I didn't miss too many people.

Most of the content was reviewed outside of this PR. I believe I've addressed all comments. If I haven't, let me know, and I'll submit follow-up PR(s).
